ui/apps/platform/src/Containers/MainPage/MainPage.tsx

```diff
@@ -57,6 +58,7 @@ function MainPage(): ReactElement {
     }
 
     const hasServiceIdentityWritePermission = hasReadWriteAccess('ServiceIdentity');
+    const isRenderedRoutePath = getIsRenderedRoutePath(hasReadAccess, isFeatureFlagEnabled);
```

Comment:
Why are we importing and using the helper function directly here, but we've wrapped in in a hook in another place?

Reply:
Good question. Deleted the hook, which is not used in these changes, because most pages will need:
* `usePermissions` and `hasReadWriteAccess for conditional rendering of buttons
* `hasReadAccess` for conditional rendering of links